action-a/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -l
+
+sh -c "echo Hello world my name is $INPUT_MY_NAME"
```

Comment:
shell scripts are handy to do common tasks required during building.  In this case, it just demonstrates your ability to pass variables between the use of your action in the workflow descriptor and here in the script file.  This allows for customization.